### PR TITLE
Add Terms Registry

### PR DESCRIPTION
**Pull request by SteveLLamb**
First pass on terms registry as per AI at prior ISDCF meeting. 

Let's focus this round of PR solely on the structure of the data, while future PR(s) can focus on updating the actual Terms, their definitions, and contents in other fields related to the Terms themselves.

After this PR is merged, will work on adding to the DCNC page. But also, ICTA (or others) may use this registry as they need on their own websites pulling from the API here.

### CODE REVIEW

**Review by palemieux — CHANGES_REQUESTED** [review]
Need to validate the URLs like at https://github.com/ISDCF/registries/blob/a92370b45a788aaac8bf59c5add1830f9c7ad09d/src/main/scripts/studios.validate.js#L39

**Comment by SteveLLamb** [comment]
One note, a thing I'm thinking to the validation, is that "relatedTerms" must be present present in the registry as a Term.

**Comment by SteveLLamb** [comment]
> Need to validate the URLs like at
> 
> > https://github.com/ISDCF/registries/blob/a92370b45a788aaac8bf59c5add1830f9c7ad09d/src/main/scripts/studios.validate.js#L39

@palemieux see https://github.com/ISDCF/registries/pull/586/commits/4650502890efc3a600916a9bdb0321d1f8d43653

**Comment by SteveLLamb** [comment]
> One note, a thing I'm thinking to the validation, is that "relatedTerms" must be present present in the registry as a Term.

addressed in https://github.com/ISDCF/registries/pull/586/commits/a76c03c8af28e9a70d49b372ad6c25e20561ddf8

File: src/main/scripts/terms.validate.js [hunk 1]
```diff
@@ -36,6 +36,22 @@ module.exports = async (registry, name) => {
...
+  const definedTerms = []
+  for (let t in registry) {
+    let term = registry[t].term
+    definedTerms.push(term)
+  }
+  for (let r in registry) {
+    let relatedTerm = registry[r].relatedTerms
+    for (let rT in relatedTerm) {
+      if (!definedTerms.includes(relatedTerm[rT])) {
+        console.log(name + " registry term '" + registry[r].term + "' contains relatedTerm '" + relatedTerm[rT] + "' that is not a defined term");
```
Comment (SteveLLamb) [inline src/main/scripts/terms.validate.js:50]: changing this to a `throw` causes build to fail

Comment (SteveLLamb) [reply src/main/scripts/terms.validate.js:50]: was trying this:
`throw `${name} registry term '${registry[r].term}' contains relatedTerm '${relatedTerm[r]}' that is not a defined term`;`

Comment (SteveLLamb) [reply src/main/scripts/terms.validate.js]: @palemieux disregard. Got it!

**Comment by SteveLLamb** [comment]
> One note, a thing I'm thinking to the validation, is that "relatedTerms" must be present present in the registry as a Term.

Done in https://github.com/ISDCF/registries/pull/586/commits/b63c868ea01b327068905993a9135e09df07a133

**Review by palemieux — APPROVED** [review]

**Comment by SteveLLamb** [comment]
To be merged after June 15th meeting if no objections.

**Pull request merged**